query/keyvault/keyvault_with_non_rbac_key_expiration_set.sql

```diff
@@ -0,0 +1,31 @@
+with non_rbac_vault as (
+  select
+    name
+  from
+    azure_key_vault
+  where not enable_rbac_authorization
+)
+select
+  -- Required Columns
+  kvk.id as resource,
+  case
+    when v.name is null then 'skip'
+    when enabled and expires_at is null then 'alarm'
+    else 'ok'
+  end as status,
+  vault_name || ' key ' || kvk.name ||
+    case
+      when v.name is null then kvk.vault_name || ' RBAC enabled vault.'
```

Comment:
Do we need to add `kvk.vault_name` in the query again since we are already addressing the vault name in `vault_name || ' key ' || kvk.name ||`?

```suggestion
      when v.name is null then || ' RBAC enabled vault.'
```